### DISCUSSION TITLE
SERIOUS ISSUE WITH LOGS

### DISCUSSION THREAD

**Discussion by Mostovm** [modrinth/.github#227] [Other discussions]
15 WHOLE GIGABYTES (15GB!) used in logs for multiple text files which had around 250MB of data EACH.
I'm confused as to what it could have possible logged AND why there is no limit to how much text it can save in a log.

I transfered files from curseforge and attempted to load minecraft with figuring out what was incompatible which would contribute to this, BUT **major issue, pls fix**.

**Comment by brawaru** [comment]
What exact logs are you talking about?
